pet-finder/src/Components/LocationInputs/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC, useState } from "react";
+
```

Comment:
These imports might be within one group: 
```
import { FC, useState } from "react";
import GooglePlacesAutocomplete from 'react-google-places-autocomplete';
```

Reply:
fixed